README.md

```diff
@@ -14,23 +12,14 @@ Start Mongo Docker container as follows:
 docker run --rm -d -p 27017:27017 --name mongo percona/percona-server-mongodb:5.0
 ```
 
-Start `<SERVICE_MANAGER_PROFILE>` services as follows:
+Start `TRACKING_CONSENT_FRONTEND` services as follows:
 
 ```bash
-sm2 --start <SERVICE_MANAGER_PROFILE>
+sm2 --start TRACKING_CONSENT_FRONTEND
```

Comment:
```suggestion
sbt run -Dplay.http.router=testOnlyDoNotUseInAppConf.Routes
```

Reply:
We need to start with the test routes for the `LegacyTestPageSpec`.

Reply:
we get that from the profile actually I think already, https://github.com/hmrc/service-manager-config/blob/main/services.json#L26868